subsys/net/ip/net_context.c

```diff
@@ -1638,6 +1638,7 @@ static int context_sendto(struct net_context *context,
 
 	pkt = context_alloc_pkt(context, len, PKT_WAIT_TIME);
 	if (!pkt) {
+		NET_ERR("Failed to allocate context");
```

Comment:
I don't think the log message is accurate? It's the net_pkt that we failed to allocate, not the net_context.

Reply:
Good point, I was reading it as `context_alloc`, not `alloc_pkt`